src/app/components/main/Main.tsx

```diff
@@ -0,0 +1,13 @@
+import * as Styled from './Main.styles';
+import { MainProps } from './Main.types';
+
+export const Main = ({ screenIsMobile, handleMobileFiltersToggle }: MainProps) => {
+  return (
+    <Styled.Main>
+      {screenIsMobile ? (
+        <Styled.Filters onClick={handleMobileFiltersToggle}>filters and categories</Styled.Filters>
```

Comment:
Dobrze by było, żeby to był jeden obiekt razem z sidebarem. Zauważ, sporo contentu Ci się powtórzy

Reply:
Rozumiem, że chodzi o to, żeby main i sidebar były jednym komponentem?

Reply:
`Styled.Filters` to komponent zawierający przycisk, który na mobilu pokazuje sidebar. Myląca nazwa, działa tak jak sugerujesz. Zmieniam nazwę tego komponentu na `FiltersSwitch`.